packages/inngest/src/components/Inngest.ts

```diff
@@ -407,21 +436,9 @@ export class Inngest<TOpts extends ClientOptions = ClientOptions> {
     let url = this.sendEventUrl.href;
 
     /**
-     * INNGEST_BASE_URL is used to set both dev server and prod URLs, so if a
-     * user has set this it means they have already chosen a URL to hit.
+     * If we're in prod mode and have no key, fail now.
```

Comment:
```suggestion
     * If in prod mode and key is not present, fail now.
```

Reply:
+1 to this clearer tone. I think we should generally avoid "we" in the SDK, so on boarding with Sylwia's suggestions!